application/server/configuration_test.go

```diff
@@ -58,6 +59,19 @@ var sampleSettings = lsp.Settings{
 	EnableSnykLearnCodeActions: "true",
 }
 
+func keyFoundInEnv(key string) bool {
```

Comment:
Wouldn't a os.getenv suffice?

Reply:
Unfortunately not, as os.getenv() is case insensitive and for PATH returns variables named "Path" and "PATH" ...